packages/@dcl/inspector/test/e2e/Hierarchy.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { App } from './pageObjects/App'
+import { Hierarchy } from './pageObjects/Hierarchy'
+import { installMouseHelper } from './utils/install-mouse-helper'
+
+describe('Hierarchy', () => {
+  beforeAll(async () => {
+    await installMouseHelper(page)
+    await page.goto('http://localhost:8000')
+    await App.waitUntilReady()
+  }, 100_000)
+
+  test('create a new entity', async () => {
+    await Hierarchy.addChild(0, 'new entity')
+    await expect(Hierarchy.exists(514)).resolves.toBe(true)
```

Comment:
I'm assuming we are using `514` as first id for a child since we are using the `mainComposite` from the feeded fs file. If that's the case, maybe we can get the number of entities in the engine in the `beforeAll` hook and use that number to compare when adding new childs to the Hierarchy. This way we avoid breaking these tests by changing the mocked composite

Reply:
good catch, I made added a `getId` helper to grab the id by the entity's label to avoid hardcoding them